modules/clients/src/test/java/org/apache/ignite/internal/processors/rest/protocols/tcp/redis/RedisProtocolConnectSelfTest.java

```diff
@@ -74,4 +75,21 @@ public void testSelect() throws Exception {
             Assert.assertEquals("v0", jedis.get("k0"));
         }
     }
+
+    //IGNITE-7153
```

Comment:
Please remove the mentioning of "IGNITE-7153". Instead, use the text below as a method description

```
/**
     * @throws Exception If failed.
     */
    @Test
```